frontend/src/components/organization/EnterBasicOrganizationInfo.js

```diff
@@ -73,6 +105,33 @@ export default function EnterBasicOrganizationInfo({
       locationKey: "location",
       texts: texts,
     }),
+    {
+      required: true,
+      label: texts.add_up_to_two_types,
+      multiselect: {
+        values: tagOptions,
+      },
+      selectedValues: selectedTypes,
+      multiSelectProps: {
+        onChange: handleChangeTypes,
+        renderValue: "",
+      },
+
+      multiple: true,
+      key: "orgtypes",
+      bottomLink: (
+        <div className={classes.selectedTypes}>
+          {selectedTypes.map((selectedType, index) => (
+            <Chip
+              className={classes.chip}
+              label={selectedType}
+              onDelete={() => handleRemoveType(selectedType)}
+              key={index}
```

Comment:
Adding a tooltip with the name for long names here would make sense

Reply:
ToolTip added.